### PR TITLE
Support main navigation

### PR DESCRIPTION
**Pull request by beerecca**

### CODE REVIEW

File: src/components/_sidepanel.scss [hunk 1]
```diff
@@ -1,76 +1,79 @@
...
-    }
+  &:focus {
+    outline: none;
+    border-color: $sb-color-blue-3;
+  }
 
-    &:hover {
-      border-color: $sb-color-blue-2;
-    }
+  &:focus:not(:focus-visible) {
```
Comment (spirosikmd) [inline src/components/_sidepanel.scss]: `focus-visible` is [not supported](https://caniuse.com/#search=%3Afocus-visible) in the browser list we mention above, right? As this code was here already, have we discussed about this?

Comment (beerecca) [reply src/components/_sidepanel.scss]: Yea, it's in Dine's original PR. It's a progressive enhancement. There's no way to achieve the equivalent without some js click tracking which we wanna avoid, so we're leaving it like this - it's a very small design preference, and it's more important to support being able to tab onto the handle easily.

Comment (spirosikmd) [reply src/components/_sidepanel.scss]: But we still keep the list of supported browsers in the comments as is, right?

Comment (beerecca) [reply src/components/_sidepanel.scss]: Yea cos it still works and looks good in those browsers, just looks the tinnnniiieeessst bit different

**Pull request merged**